slick/src/main/scala/slick/jdbc/SQLiteProfile.scala

```diff
@@ -98,14 +98,16 @@ trait SQLiteProfile extends JdbcProfile {
     override def createColumnBuilder(tableBuilder: TableBuilder, meta: MColumn): ColumnBuilder = new ColumnBuilder(tableBuilder, meta) {
       /** Regex matcher to extract name and length out of a db type name with length ascription */
       final val TypePattern = "^([A-Z]+)(\\(([0-9]+)\\))?$".r
+      final val TimestampPattern = """^"(.*)"""".r
```

Comment:
Is this sufficient to cover all time strings allowed by SQLite (See: https://www.sqlite.org/lang_datefunc.html section "Time Strings")? For example, SQLite allows the following time string "2456572.84952685", while `java.sql.Timestamp` expects a string in format "yyyy-[m]m-[d]d hh:mm:ss[.f...]".